src/containers/SinglePost.js

```diff
@@ -22,7 +24,26 @@ SinglePost.propTypes = {
 
 export default compose(
   prefetch(({ dispatch, params }) => join(
-    dispatch(fetchPost(params.id)),
+    dispatch(fetchPost(params.id))
+    .then(action => {
+      let payload = action.payload
+      if (payload && !payload.api) {
+        let post = payload
+        let { media } = post
+        var metaTags = {
+          'og:title': post.name,
+          'og:description': truncate(striptags(post.description || ''), 140)
+        }
+        if (media[0]) {
+          metaTags = {...metaTags, ...{
```

Comment:
also, it's a bit of a toss-up, but there's no need to treat this as immutable, so it'd be equally valid to use `lodash.extend` or something